_events/sr2021/league-3.md

```diff
@@ -4,7 +4,20 @@ date: 2021-03-20
 time_tbc: true
 layout: event
 type: competition
-location: Student Robotics' YouTube Channel
+location: <a href="https://www.youtube.com/watch?v=MpuhtW4mCKM">Student Robotics' YouTube Channel</a>
 ---
 
 This is the third league round of this year's competition and we'll be able to see the result of a few months of robot iteration! Our teams will work hard to push their virtual robots to the limit, competing against each other, and evolving their strategies to gain the upper hand.
+
+Code submission will be due at 8pm UK time on Friday 19th: [Submit your code](https://studentrobotics.org/code-submitter/)
+
+Our live-stream will start at noon UK time: [Watch live](https://www.youtube.com/watch?v=MpuhtW4mCKM)
```

Comment:
Please could we be specific about the timezone here? If it's not GMT we should say explicitly that it's BST. People outside the UK (even in Europe) likely won't know exactly when our clocks change.

Even better would be to use a `<time>` element with a suitable `datetime` attribute to specify the instant precisely. It _may_ be worth extending our event-times JS stuff to being sufficiently general to add a dynamic title to such `<time>` elements, though maybe not worth it for now? (I suspect that the limitations of the translation handling maybe such that  in the context of prose having it say "8pm BST" for everyone is preferable to trying to swap in something location specific, but that may be worth exploring if we have time).